plugins/jvm/ts/jvmHelpers.ts

```diff
@@ -100,16 +100,7 @@ namespace JVM {
     //must save to local storage, to be picked up by new tab
     saveConnection(clone);
     const $window: ng.IWindowService = HawtioCore.injector.get<ng.IWindowService>('$window');
-    var url
-    if(clone.view) {
```

Comment:
I'm not sure if this code is ever called, as clone is extended from connection options that does not have view. But since I do not know how to test it. Leave is as is